infinitest-eclipse/src/main/java/org/infinitest/eclipse/prefs/SwtColorFieldEditor.java

```diff
@@ -0,0 +1,33 @@
+package org.infinitest.eclipse.prefs;
+
+import org.eclipse.jface.preference.*;
+import org.eclipse.swt.*;
+import org.eclipse.swt.widgets.*;
+
+public class SwtColorFieldEditor extends ComboFieldEditor {
```

Comment:
Ok I understand This should be doable using org.eclipse.jface.resource.ColorRegistry. 
However I'm fine with current implementation.